### PR TITLE
chore(workflow-v2.7.0): releasing workflow-v2.7.0(-e2e)

### PR DESCRIPTION
**Pull request by mboersma**

### CODE REVIEW

**Comment by jchauncey** [comment]
if you want to test upgrading here ya go - https://deis.com/docs/workflow/managing-workflow/upgrading-workflow/

**Comment by jchauncey** [comment]
@mboersma we should bring over the changes that are in #371

**Comment by mboersma** [comment]
Updated to sync up with #371.

File: workflow-v2.7.0/tpl/deis-database-service.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: deis-database
+  namespace: deis
+  labels:
+    heritage: deis
+  amnotations:
```
Comment (vdice) [inline workflow-v2.7.0/tpl/deis-database-service.yaml]: s/amnotations/annotations

Comment (jchauncey) [reply workflow-v2.7.0/tpl/deis-database-service.yaml]: good catch. we should maek sure that same spelling error isnt in @kmala 's pr

Comment (jchauncey) [reply workflow-v2.7.0/tpl/deis-database-service.yaml]: just checked and its not.

Comment (mboersma) [reply workflow-v2.7.0/tpl/deis-database-service.yaml]: My bad--fixed!

**Pull request merged**